docs/_layouts/blog-page.html

```diff
@@ -5,14 +5,25 @@
     <header>
         <h1>{{ page.title }}</h1>
         <div class="byline">
-            <time class="date">
-                <i class="far fa-calendar-alt"></i>
-                {{ page.date | date: "%Y-%m-%d" }}
-            </time>
-            <span class="author">
-                <i class="fas fa-pen-nib"></i>
-                {{ page.author }}
-            </span>
+            {% if page.author and page.authorImg %}
+            <img id="author-img" src="{{ site.baseurl }}{{ page.authorImg }}">
+            {% endif %}
+            <div class="secondary-infos">
+                {% if page.author and page.authorImg %}
```

Comment:
Some pages have author(s) specified without images, such as: https://dotty.epfl.ch/blog/2015/10/23/dotty-compiler-bootstraps.html
So I think it would be better to check just for author here (`{% if page.author %}`)